src/include/binder/expression/property_expression.h

```diff
@@ -40,6 +44,7 @@ class PropertyExpression : public Expression {
     }
 
 private:
+    bool isPrimaryKey_ = false;
```

Comment:
Why do we need the `_` postfix?

Reply:
because it conflicts with get function name